src/main/java/com/boomi/flow/services/boomi/mdh/database/FieldMapper.java

```diff
@@ -177,7 +177,11 @@ public static String getUniqueIdByPropertyName(String modelName, Property proper
 
     static Object createMapEntry(Property property, String modelName, List<Universe.Layout.Model.Element> elements) {
         if (property.getContentValue() != null) {
-            switch (property.getContentType()) {
+            switch (
+                property.getContentType() != null ?
+                    property.getContentType() :
+                    ContentType.String
+            ) {
```

Comment:
@TomFullalove this solution works, but It is a bit hard to read and it says that the default behavior should be stringContent and that could induce to an error in the future. Could you change it to `if` or add a comment to those lines.